localstack-core/localstack/services/kms/provider.py

```diff
@@ -1354,6 +1357,74 @@ def enable_key_rotation(
             key.rotation_period_in_days = request.get("RotationPeriodInDays")
         key._update_key_rotation_date()
 
+    @handler("ListKeyRotations", expand=False)
+    def list_key_rotations(
+        self, context: RequestContext, request: ListKeyRotationsRequest
+    ) -> ListKeyRotationsResponse:
+        # https://docs.aws.amazon.com/kms/latest/developerguide/key-state.html
+        # "If the KMS key has imported key material or is in a custom key store: UnsupportedOperationException."
+        # We do not model that here, though.
+        # TODO: Add support for single-region EXTERNAL key origins
+        key = self._get_kms_key(
+            context.account_id,
+            context.region,
+            request.get("KeyId"),
+            disabled_key_allowed=True,
+            pending_deletion_key_allowed=True,
+        )
+
+        include_key_material = request.get("IncludeKeyMaterial")
+        if include_key_material is not None:
+            if include_key_material not in ["ALL_KEY_MATERIAL", "ROTATIONS_ONLY"]:
```

Comment:
`ALL_KEY_MATERIAL` and `ROTATIONS_ONLY` should be used from the `IncludeKeyMaterial` class in `__init__.py` file which already includes the types you've defined here. 

(Note: This is applicable in other changes made to this PR, I am not commenting in every place. Let me know if there is any confusion around this, happy to help and explain more 🙂)